packages/inline-definition/README.md

```diff
@@ -52,10 +52,11 @@ npm install @leafygreen-ui/inline-definition
-| ------------ | ------------------------------ | ------------------------------------------------------------ | ------- |
-| `definition` | **Required** `React.ReactNode` | Content that describes the term. Will appear inside Tooltip. |         |
-| `children`   | `string`                       | Text that will appear underlined                             |         |
-| `className`  | `string`                       | className will be applied to the trigger element             |         |
-
-_Any other properties will be spread on the `Tooltip` element_
+| Prop                    | Type                                           | Description                                                                                                                                                                                                                                        | Default   |
+| ----------------------- | ---------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- |
+| `definition` (Required) | `React.ReactNode`                              | Content that describes the term. Will appear inside Tooltip.                                                                                                                                                                                       |           |
+| `children`              | `string`                                       | Text that will appear underlined                                                                                                                                                                                                                   |           |
+| `className`             | `string`                                       | className will be applied to the trigger element                                                                                                                                                                                                   |           |
+| `align`                 | `'top'` \| `'bottom'` \| `'left'` \| `'right'` | Determines the preferred alignment of the `<Tooltip />` component relative to the element passed to the `trigger` prop. If no `trigger` is passed, the Tooltip will be positioned against its nearest parent element.                              | `'top'`   |
+| `justify`               | `'start'` \| `'middle'` \| `'end'` \| `'fit'`  | Determines the preferred justification of the `<Tooltip />` component (based on the alignment) relative to the element passed to the `trigger` prop. If no `trigger` is passed, the Tooltip will be positioned against its nearest parent element. | `'start'` |
```

Comment:
these are still referring to `Tooltip`. also `trigger` is hard-coded in `InlineDefinition`, so there is no trigger prop.

Reply:
👍